src/plugins/discourse/mirrorRepository.js

```diff
@@ -293,6 +313,27 @@ export class SqliteMirrorRepository
     return toAddResult(res);
   }
 
+  topicsInCategories(
+    categoryIds: $ReadOnlyArray<CategoryId>
+  ): $ReadOnlyArray<TopicId> {
+    return this._db
+      .prepare(
+        dedent`\
+          SELECT id FROM topics
+          WHERE category_id IN (${categoryIds.map((_) => "?").join(",")})
```

Comment:
This pattern feels a little funky to me--surely there should be a way to put an array in that doesn't require manually creating a query with array.length "?" characters? cc @wchargin. That said, I don't see an actual problem with it, so I'll approve

Reply:
I thought so too. But I've not seen any option to bind arrays with better-sqlite3 in their API.
The flow type also hints at this, being either a dictionary for named parameters or a positional value defined as `number | string | Buffer | null`.